frontend/src/components/intake/ShasIntakeForm.vue

```diff
@@ -161,9 +171,23 @@ onBeforeMount(async () => {
               :click-callback="clickCallback"
               title="Basic info"
               icon="fa-user"
+              :class="{
+                'p-red':
```

Comment:
The `p-` prefix is a primevue directive. We should not be using it in our own code. `error-red` or something would be a better fit.